src/app/components/HoldRequest/HoldRequest.jsx

```diff
@@ -36,9 +36,8 @@ class HoldRequest extends React.Component {
   }
```

Comment:
I think it was done to have the function return something. It's not necessary I guess, but good to have. The page will redirect anyways. It's also a good way to tell the function it's done.